test/unit/test_model_server.py

```diff
@@ -27,13 +30,18 @@
 @patch('subprocess.call')
 @patch('subprocess.Popen', return_value=MMS_PROCESS)
 @patch('sagemaker_inference.model_server._add_sigterm_handler')
+@patch('sagemaker_inference.model_server._install_requirements')
+@patch('os.path.exists', return_value=True)
 @patch('sagemaker_inference.model_server._create_model_server_config_file')
 @patch('sagemaker_inference.model_server._adapt_to_mms_format')
-def test_start_model_server_default_service_handler(adapt, create_config, sigterm, subprocess_popen, subprocess_call):
+def test_start_model_server_default_service_handler(adapt, create_config, exists, install_requirements, sigterm,
+                                                    subprocess_popen, subprocess_call):
     model_server.start_model_server()
 
     adapt.assert_called_once_with(model_server.DEFAULT_HANDLER_SERVICE)
     create_config.assert_called_once_with()
+    exists.assert_called_once_with(REQUIREMENTS_PATH)
+    install_requirements.assert_called_once_with()
```

Comment:
nit/optional: since `_install_requirements()` doesn't take any arguments, there's not really a need to check that it was called with no arguments (otherwise there would be a runtime error), and so `assert_called_once()` would also suffice here. It just looks a little funny to me to have "with" followed by nothing, so it jumped out at me. Not strongly opposed to it as is though.